src/main/resources/english.yml

```diff
@@ -113,9 +113,9 @@ msg_war_siege_peaceful_player_punished_for_being_in_siegezone: '&cWar-allergy ac
 msg_war_siege_err_cannot_attack_peaceful_town: '&cYou cannot attack a peaceful town.'
 
 #Siege immunities
-msg_set_siege_immunities_town: 'Siege immunity for town %s set to %s hours.'
-msg_set_siege_immunities_nation: 'Siege immunities for all towns in nation %s set to %s hours.'
-msg_set_siege_immunities_all: 'Siege immunities for all towns set to %s hours.'
+msg_set_siege_immunities_town: 'Siege immunity for town %s set to %s.'
+msg_set_siege_immunities_nation: 'Siege immunities for all towns in nation %s set to %s.'
+msg_set_siege_immunities_all: 'Siege immunities for all towns set to %s.'

```

Comment:
:construction: 
- Apologies, I mis-typed something in the last review:
- Please also move the **modified** lines down to the bottom of the language files
- This is particularly so that the translators of the existing French file, will more easily see that they have to change their translation.